sdk/playwrighttesting/microsoft-playwright-testing/src/common/constants.ts

```diff
@@ -33,6 +33,7 @@ export const ServiceAuth = {
 export const ServiceEnvironmentVariable = {
   PLAYWRIGHT_SERVICE_OS: "PLAYWRIGHT_SERVICE_OS",
   PLAYWRIGHT_SERVICE_RUN_ID: "PLAYWRIGHT_SERVICE_RUN_ID",
+  PLAYWRIGHT_SERVICE_RUN_NAME: "PLAYWRIGHT_SERVICE_RUN_NAME",
```

Comment:
Use Internal environment variables, we don't want to expose this to the customer